### DISCUSSION TITLE
Hide status column when filtering

### DISCUSSION THREAD

**Discussion by seanmtaylor** [community/community#5881] [Projects and Issues] [answered]
We have a default filter of `-status:Planning` from our main sprint Board view.

However, the empty column for this status still shows.

![Screenshot from 2021-09-29 13-59-54](https://user-images.githubusercontent.com/1545704/135273489-ba53794d-ab3c-4a37-8057-80c1c45f8261.png)

Would it be possible to hide status columns if they are excluded in the filters?

**Comment by rileybroughten** [comment, marked as answer]
This has been released! Check out this [similar post](https://github.com/orgs/community/discussions/6906#discussioncomment-3538869) with the announcement, as you can filter out board columns and hide them directly from the view 👍
